cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/BrokerFailureDetector.java

```diff
@@ -177,12 +181,22 @@ private void parsePersistedFailedBrokers(String failedBrokerListString) {
     }
   }
 
+  private boolean tooManyFailedBrokers(int deadBrokerCount, int aliveBrokerCount) {
```

Comment:
Nit: For consistency with the relevant config: `deadBrokerCount` -> `failedBrokerCount`?